madgraph/interface/reweight_interface.py

```diff
@@ -1712,6 +2208,8 @@ def create_standalone_directory(self, second=False):
             path_me = self.me_dir
         else:
             path_me = self.rwgt_dir
+            ## TV: skip all output and generation of folder is rw_me exists already
+            return
```

Comment:
This confuse me because you do not check if self.rwgt_dir exists or not .... The first time it will be set and need to be created). Did you check this carefully (also in the non sudakov case)?

Reply:
Hi Marco, this comment might still need some check... (I still found the way to scan each non solved comment) --so this is line 1979 of reweight_interface)